components/authz-service/server/v2/authz.go

```diff
@@ -26,15 +30,25 @@ type authzServer struct {
 	engine   engine.V2Authorizer
 	vSwitch  *VersionSwitch
 	projects api.ProjectsServer
+	store    storage.Storage
 }
 
-// NewAuthzServer returns a new IAM v2 Authz server.
-func NewAuthzServer(l logger.Logger, e engine.V2Authorizer, v *VersionSwitch, p api.ProjectsServer) (api.AuthorizationServer, error) {
+// NewPostgresAuthzServer returns a new IAM v2 Authz server.
+func NewPostgresAuthzServer(l logger.Logger, e engine.V2Authorizer, v *VersionSwitch, p api.ProjectsServer) (api.AuthorizationServer, error) {
+	s := postgres.GetInstance()
+	if s == nil {
+		return nil, errors.New("postgres v2 singleton not yet iniitalized for authz server")
```

Comment:
Need a global search/replace--this occurs a few times 😁 

s/iniitalized/initialized/